build.gradle

```diff
@@ -206,6 +217,19 @@ subprojects {
       } 
     }
   }
+
+  publishing {
+     publications {
+        mavenJava(MavenPublication) {
+	   groupId "org.apache.edgent"
+           artifactId "$project.group".replace("edgent.", "") + "." + artifact
+	   version build_version
+           artifact sourceJar
+
+	   from components.java
+        }
+     }
+  }
```

Comment:
and I think the next step that we all look farward to is to make Edgent go into Maven Central Repository :)